.github/workflows/deploy.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+name: deploy
+on:
+  push:
+    tags:
+      - "v*"
```

Comment:
Can we prefix with `0` instead? I think the only tags will be for releases, but we can at least filter on the 0.x.y tags. When we hit 1.x.y, we can revisit if releasing on every tag makes sense or not.

Reply:
Thanks for the feedback @srchase, I've pushed a commit to change it from `v*` to `0*` as you've requested :)